build/azure-pipelines/linux/sql-product-build-linux.yml

```diff
@@ -149,11 +149,16 @@ steps:
     continueOnError: true
     condition: and(succeeded(), eq(variables['RUN_UNSTABLE_TESTS'], 'true'))
 
-  - script: |
+  - bash: |
```

Comment:
what's the benefit of switching to bash?

Reply:
the script I added below is bash script, I guess it doesn't matter since by default the script is bash on Linux.